subset/hold/test_hold

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
```

Comment:
should the old (docker/include/bin) test_hold be deleted?

Reply:
It's used in building of another container.